samples/module_controller_pod.yaml

```
@@ -25,7 +27,19 @@ spec:
-          value: YOUR_ENV
\ No newline at end of file
+        - name: ENV # module controller env, will set to vnode labels
+          value: YOUR_ENV
+        - name: IS_CLUSTER # cluster flag, if true, will use cluster configure to start virtual kubelet
+          value: "false"
+        - name: WORKLOAD_MAX_LEVEL # cluster configure, means max workload level for workload calculation in virtual kubelet, default is 3
+          value: "3"
+        - name: ENABLE_MODULE_DEPLOYMENT_CONTROLLER # module deployment controller flag, if true, will start deployment controller to modify module deployment's replicas and baseline
+          value: "true"
+        - name: VNODE_WORKER_NUM # VNode concurrent module processing thread number, set to 1 to indicate single thread
+          value: "8"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)